### PR TITLE
Removed client and hubactor abstractions, bugfix, cleanup

### PR DESCRIPTION
**Pull request by chris-ray**

### CODE REVIEW

File: src/examples/Hello World/Server/Program.cs [hunk 1]
```diff
@@ -37,21 +48,26 @@ protected override void OnStarted()
...
+                Console.WriteLine($"Client {clientId} Joined.");
         }
 
-        protected override void OnClientLeft(string clientId)
+        private void ClientLeft(string clientId)
         {
-            Console.WriteLine($"client left: {clientId}");
-            _clients.Remove(clientId);
+            if (_clients.Remove(clientId))
+                Console.WriteLine($"Client {clientId} Joined.");
```
Comment (marcpiechura) [inline src/examples/Hello World/Server/Program.cs]: Joined need to be left ;)

**Comment by marcpiechura** [comment]
:+1: could you also squash the commits ?

**Comment by chris-ray** [comment]
Ok I've never done a squash before so tell me if I did it wrong. Not sure why it says it has conflicts...

**Comment by chris-ray** [comment]
alright hold on, I'm working on it :) git can be confusing sometimes

**Comment by chris-ray** [comment]
Ok I think that did it. sorry for the mess.

**Pull request merged**